pages/common/cargo.md

```diff
@@ -15,13 +15,13 @@
 
 `cargo install --list`
 
-- Create a new binary Rust project in the current directory:
+- Create a new binary or library Rust project in the current directory:
 
-`cargo init --bin`
+`cargo init --{{bin|lib}}`
 
-- Create a new library Rust project in the current directory:
+- Create a new binary or library Rust project in the specified directory:
```

Comment:
But you are not specifying any directory. Typo ?

Reply:
Sorry, `crate_name` should have been `path`